tests/Silex/Tests/Provider/SpoolStub.php

```diff
@@ -36,7 +36,7 @@ public function isStarted()
 
     public function queueMessage(\Swift_Mime_Message $message)
     {
-        $this->messages[] = $message;
+        $this->messages[] = clone $message;
```

Comment:
Without this, the new `testSwiftMailerSenderAddress` fails.

[Swift_Plugins_ImpersonatePlugin](https://github.com/swiftmailer/swiftmailer/blob/master/lib/classes/Swift/Plugins/ImpersonatePlugin.php) reverts the headers after the message has been sent (in the `sendPerformed` event), so without cloning, `$message->getReturnPath()` would not return the value used when the message is sent.